src/router/routes.tsx

```diff
@@ -464,6 +463,86 @@ export const { routes, reactRouterDOMRoutes, hooks, components, utils } = new In
     hideSideNav: false,
     authLevels: ['admin', 'support'],
   })
+  .addRoute({
+    key: 'PROVIDE_ESERVICE_TEMPLATE_CATALOG',
+    path: '/erogazione/catalogo-template',
+    element: <ProviderEServiceTemplatesCatalogPage />,
+    public: false,
+    hideSideNav: false,
+    authLevels: ['admin', 'support'],
```

Comment:
From here to the bottom:
```suggestion
    authLevels: ['admin', 'api', 'support'],
```